pom.xml

```diff
@@ -602,6 +602,19 @@
       <properties>
         <hadoop.version>1.0.4</hadoop.version>
       </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <excludes>
+                <exclude>**/AlluxioFileSystem.java</exclude>
+              </excludes>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
```

Comment:
@calvinjia, a better solution is to compute compiling/testing policy on the fly with user-provided hadoop version.